src/fs/protocol/private/namefilter.ts

```diff
@@ -61,7 +61,7 @@ export const toHash = async (filter: BloomFilter): Promise<PrivateName> => {
 }
 
 // saturate a filter (string) to 320 bits
-export const saturate = async (filter: RevisionNameFilter, threshold = SATURATION_THRESHOLD): Promise<SaturatedNameFilter> => {
+export const saturate = async (filter: RevisionNameFilter | BareNameFilter, threshold = SATURATION_THRESHOLD): Promise<SaturatedNameFilter> => {
```

Comment:
I didn't quite figure out where this was needed from reading the code. Can this be changed back?

Reply:
Will change back, ended up not using this.